python/pyspark/cloudpickle_fast.py

```diff
@@ -0,0 +1,557 @@
+"""
```

Comment:
You could add cloudpickle_fast.py to `dev/.rat-excludes` to avoid the RAT test error.